lib/migration.c

```diff
@@ -56,24 +56,44 @@ struct migration {
     } iter;
 };
 
+struct migr_state_data {
+    __u32 state;
```

Comment:
Not for this, but why are we using __ or indeed u* at all? We're in userspace without any namespacing issues; we can/should use the formal uint32_t types